manifests/ray-serve-llama/model_deployment/base/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: llama2-server-controller
+  namespace: opendatahub
+  labels:
+    app: llama2-server-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: llama2-server-controller
+  template:
+    metadata:
+      labels:
+        app: llama2-server-controller
+    spec:
+      containers:
+      - name: ray-odh-tests
+        image: quay.io/michaelclifford/deep-thought:v0.0.5
```

Comment:
why is this image required?

Reply:
@Shreyanand  This is the image that actually runs the Ray code to deploy the service. the code can be found in  `serve_llama2.py`. I admit this is a bad name for it and we can rename it to something better next time we rebuild the image.